update-bom/entrypoint.sh

```diff
@@ -40,13 +42,38 @@ then
     done
 fi
 
+if [ -f $CATALOG_FILE ]; then
+    echo "Updating version catalog..."
+    if [ ! -z $bomProperty ]; 
+    then
+        catalogProperty=$(echo $bomProperty | sed -r 's/([a-z0-9])([A-Z])/\1-\L\2/g' | sed -r 's/-version$//g')
+        echo "name: $catalogProperty"
+        sed -i -E "s/^(managed-)?$catalogProperty\s*=\s*\".*$/\1$catalogProperty \= "${projectVersion}"/" $CATALOG_FILE
```

Comment:
Don't you need to add here `echo "value: ${!catalogProperty}"`?

Reply:
This is AFAIU only for debugging. It should actually be "${!bomProperty}" but this isn't used since we use the project version. Anyway I updated the PR with this change, thanks!